salt/states/dracr.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+'''
+.. versionadded:: 2018.3.2
+
+Management of Dell DRAC
+
+The DRAC module is used to create and manage DRAC cards on Dell servers
+
+Ensure the property is set
+
+  .. code-block:: yaml
+
+  test:
+    dracr.property:
+       - properties:
+           System.ServerOS.HostName: "Pretty-server"
+           System.ServerOS.OSName: "Ubuntu 16.04"
+       - admin_password: calvin
+       - admin_root: myuser
+       - host: 10.10.10.1
+
+'''
+
+from __future__ import absolute_import
+
+import salt.exceptions
+
+
+def __virtual__():
+    '''
+    Ensure the racadm command is installed
+    '''
+    if salt.utils.which('racadm'):
+        return True
+
+    return False
+
+
+def property(properties, admin_username='root', admin_password='calvin', host=None, **kwargs):
```

Comment:
What about `property_present` instead of just `property`? I think that is more inline with our state syntax.